plugin/vscode/extension.js

```diff
@@ -426,6 +437,9 @@ async function deactivateAsync() {
   let disposing = toDispose.splice(0, toDispose.length);
   disposing.reverse();
   for (let disposable of disposing) {
+    if (disposable instanceof(DocumentLinterCollection)) {
+      disposable._performanceWriter.writeToFile()
+    }
```

Comment:
Why not have DocumentLinterCollection#dispose call `this._performanceWriter.writeToFile()`?

Reply:
Used suggestion in latest commit